cpp_guidelines/.clang-format

```diff
@@ -0,0 +1,107 @@
+---
+Language: Cpp
+Standard: c++17
+DisableFormat:   false
+AccessModifierOffset: -4
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
+AlignConsecutiveBitFields: true
+AlignConsecutiveDeclarations: false
+AlignConsecutiveMacros: true
+AlignEscapedNewlines: Right
+AlignOperands: true
```

Comment:
VOTE

current | false |
--------|-------------|
 0      |   1         |

Alignment always results into the issue that one line change can trigger re-alignment of other lines and will increase the diff.

Reply:
current | DontAlign
-- | --
0 | 2

Reply:
current | DontAlign
-- | --
0 | 3

Reply:
current | DontAlign |
----------|---------------|
1 | 3

Reply:
Voting closed, the winner is `DontAlign`.